MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -1048,6 +1021,10 @@ private bool HasChanged(BaseItem item, IHasItemChangeMonitor changeMonitor, IDir
                 {
                     target.Studios = source.Studios;
                 }
+                else if (source.Studios.Length >= 0)
```

Comment:
If it's equal to 0, it doesn't change anything?

Reply:
Since we're oly in this check if we either not replace or the target already has studios this is intended. If souce has studios, we append, if it does not we ignore.

Reply:
I was just wondering why the check was `>= 0` and not `> 0`

Reply:
I think removing the checks to delete any duplicates is sensible regardless